src/python/grpcio/grpc/_cython/_cygrpc/common.pyx.pxi

```diff
@@ -0,0 +1,45 @@
+# Copyright 2023 The gRPC Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+class BaseError(Exception):
+    """
+    This class is EXPERIMENTAL.
```

Comment:
Is your plan to merge this immediately and then remove the experimental docstrings in 2 weeks? Or do you want to hold off on merging this for 2 weeks? It's up to you.

Reply:
Plan to merge it after 2 weeks, I'll remove those `EXPERIMENTAL`.